frontend/main.js

```diff
@@ -266,8 +271,28 @@ async function startApp () {
           'times-circle'
         )
       }
+
+      this.setBadgeOnTab()
+
+      this.$el.addEventListener('mousemove', this.requestPermissions, false)
```

Comment:
We should request notification permissions not when they load the app, but when the user has entered a group (e.g. when the Group Dashboard becomes visible). This way they can get some context for why the permissions are necessary, and they will be already somewhat invested in receiving them - that way they are less likely to reflexively say "No" when they don't mean to.

Reply:
Make sense.